ci-operator/config/openshift/installer/openshift-installer-release-4.17__okd-scos.yaml

```diff
@@ -0,0 +1,158 @@
+  from: origin_scos-4.17_base-stream9
+  inputs:
+    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      as:
+      - builder
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+    origin_scos-4.17_installer-etcd-artifacts:
+      as:
+      - etcd-artifacts
+      - registry.ci.openshift.org/ocp/4.17:installer-etcd-artifacts
+    origin_scos-4.17_installer-kube-apiserver-artifacts:
+      as:
+      - kas-artifacts
+      - registry.ci.openshift.org/ocp/4.17:installer-kube-apiserver-artifacts
+  to: installer-altinfra
+- build_args:
+  - name: TAGS
+    value: scos
+  dockerfile_path: images/installer-artifacts/Dockerfile.rhel
+  from: origin_scos-4.17_base-stream9
+  inputs:
+    installer-terraform-providers:
+      as:
+      - providers
+      - registry.ci.openshift.org/ocp/4.17:installer-terraform-providers
+    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      as:
+      - builder
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+    origin_scos-4.17_installer-etcd-artifacts:
+      as:
+      - etcd-artifacts
+      - registry.ci.openshift.org/ocp/4.17:installer-etcd-artifacts
+    origin_scos-4.17_installer-kube-apiserver-artifacts:
+      as:
+      - kas-artifacts
+      - registry.ci.openshift.org/ocp/4.17:installer-kube-apiserver-artifacts
+  to: installer-artifacts
+- build_args:
+  - name: TAGS
+    value: scos
+  dockerfile_path: images/infrastructure-providers/Dockerfile
+  from: origin_scos-4.17_base-stream9
+  inputs:
+    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      as:
+      - linuxarmbuilder
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+  to: installer-terraform-providers
+- build_args:
+  - name: TAGS
+    value: scos
+  dockerfile_path: images/installer/Dockerfile.ci
+  from: origin_scos-4.17_base-stream9
+  inputs:
+    installer-terraform-providers:
+      as:
+      - providers
+      - registry.ci.openshift.org/ocp/4.17:installer-terraform-providers
+    ocp_builder_rhel-9-golang-1.22-openshift-4.17:
+      as:
+      - builder
+      - registry.ci.openshift.org/ocp/builder:rhel-9-golang-1.22-openshift-4.17
+    origin_scos-4.17_installer-etcd-artifacts:
+      as:
+      - etcd-artifacts
+      - registry.ci.openshift.org/ocp/4.17:installer-etcd-artifacts
+    origin_scos-4.17_installer-kube-apiserver-artifacts:
+      as:
+      - kas-artifacts
+      - registry.ci.openshift.org/ocp/4.17:installer-kube-apiserver-artifacts
+  to: installer
+promotion:
+  to:
+  - disabled: true
+    name: scos-4.17
+    namespace: origin
+releases:
+  latest:
+    integration:
+      name: scos-4.17
+      namespace: origin
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: e2e-aws-ovn
+  optional: true
+  run_if_changed: (scos|fcos|okd)
+  steps:
+    cluster_profile: aws
+    workflow: openshift-e2e-aws
+- as: e2e-vsphere-ovn
```

Comment:
The `openshift-installer-master__okd-scos.yaml` pushes to the scos-4.17 stream with the same configs - we need to add the vSphere job there as well. Do we need this 4.17 file? i thought master would suffice till we GA (it would effectively be the same)

Reply:
I just followed the normal configs, that was the pattern

Reply:
@Prashanth684 I removed 4.17 and changed master to include vsphere. Going to retest with master. Somethign strange is going on with the workflow in the last run